CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.22.0]
+
+### Changed
+
+- Update `microkelvin` from `0.6` to `0.10` #158
+- Update `nstack` from `0.9` to `0.10` #158
```

Comment:
Add the square bracket otherwise it's not a valid markdown link (e.g. `[#158]`).

Also, add the full link at the bottom of the CHANGELOG: it seems this CHANGELOG doesn't contain the proper links in the footer (see [dusk-abi's changelog](https://raw.githubusercontent.com/dusk-network/dusk-abi/master/CHANGELOG.md) as example)

Reply:
Fixed